chapters/540_data.md

```diff
@@ -153,15 +153,14 @@ __[List 02: relevant (sub-)categories of data]{#list-02_data-categories}__
 The available *primitives* mainly depend on those who are supported by the query language itself.
 In this case, all *primitives* mentioned above are supported by *SPARQL* [@web_spec_xml_types] and 
 *GraphQL* [@web_spec_graphql_types]. When choosing a database system it has to be ensured that 
-either the system already supports the required *primitives* or they can be emulated somehow with a 
-least amount of drawbacks. When modelling relations between data points one can use for example keys 
-(or identifiers) to make reference, or additional syntactical tools like *lists* (or arrays) and 
+either the system already supports the required *primitives* or they can be emulated somehow minimal drawbacks. When modelling relations between data points, one can use, for example, keys 
```

Comment:
I changed "be emulated somehow minimal drawbacks" to "be emulated somehow with minimal drawbacks". if thats not correct, pls open a PR including a short explanation